src/main/java/cz/fi/muni/pa165/hockeymanager/entity/User.java

```diff
@@ -41,14 +42,22 @@
     @Setter
     private Role role = Role.PLAYER;
 
-    @OneToOne
+    @OneToOne(fetch= FetchType.LAZY)
     @Getter
     @Setter
     private Team team;
 
     public User() {
     }
 
+    public User(String name, String email, String password, Role role) {
+        this.name = name;
+        this.email = email;
+        this.password = password;
+        this.role = role;
+        this.team = null;
+    }
```

Comment:
code copy => this(name, email, password, role, null) - use the constructor that is already available